src/main/java/db/Storage.java

```diff
@@ -0,0 +1,11 @@
+package db;
+
+import model.Product;
+import model.User;
+
+import java.util.ArrayList;
+
+public class Storage {
+    public static final ArrayList<User> USERS = new ArrayList();
```

Comment:
```suggestion
    public static final ArrayList<User> users = new ArrayList();
```
because it is mutable object :)